server/src-lib/Hasura/Server/Telemetry.hs

```diff
@@ -189,11 +189,11 @@ computeActionsMetrics ac ao =
         asyncActionsLen  = length . filter ((==(ActionMutation ActionAsynchronous)) . _adType . _aiDefinition) $ actions
         queryActionsLen = length . filter ((==ActionQuery) . _adType . _aiDefinition) $ actions
 
-        outputTypesLen = length . nub . (map (_adOutputType . _aiDefinition)) $ actions
-        inputTypesLen = length . nub . concat . (map ((map _argType) . _adArguments . _aiDefinition)) $ actions
+        outputTypesLen = length . L.nub . (map (_adOutputType . _aiDefinition)) $ actions
+        inputTypesLen = length . L.nub . concat . (map ((map _argType) . _adArguments . _aiDefinition)) $ actions
         customTypesLen = inputTypesLen + outputTypesLen
 
-        typeRelationships = length . nub . concat . map ((getActionTypeRelationshipNames ao) . _aiDefinition) $ actions
+        typeRelationships = length . L.nub . concat . map ((getActionTypeRelationshipNames ao) . _aiDefinition) $ actions
```

Comment:
This section looks like it could use some refactoring as it is very repetitive, but it's good that you didn't do that in this PR.